test/parallel/test-fs-realpath.js

```diff
@@ -197,7 +197,7 @@ function test_cyclic_link_protection(callback) {
     fs.symlinkSync(t[1], t[0], 'dir');
     unlink.push(t[0]);
   });
-  assert.throws(function() { fs.realpathSync(entry); }, /ELOOP/);
+  assert.throws(function() { fs.realpathSync(entry); });
```

Comment:
@bzoz Sorry, you’re right, that second `test_deep_symlink_eloop` part does not work with the JS implementation as-is. I think it can be fixed up to be included here, but that’s probably something better kept for another PR.